include/veriblock/base59.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef _BASE59_H
+#define _BASE59_H
+
+#include <string>
+#include <vector>
+
+namespace VeriBlock {
+
+    std::string EncodeBase59(const unsigned char *pbegin,
+        const unsigned char *pend);
+
+    // std::string EncodeBase58(const uint8_t *buf, size_t size);
```

Comment:
why commented?

Reply:
header has been copied from your base58.hpp, but 58 - postfix was changed to 59 - postfix. 
Delete this comments on base58 and base59?

Reply:
uncommented this function